src/mrpro/data/_DcfData.py

```diff
@@ -193,3 +197,9 @@ def from_traj_voronoi(cls, traj: KTrajectory) -> DcfData:
             dcf = torch.ones(*traj.broadcasted_shape[-3:], 1, 1, 1, device=traj.kx.device)
 
         return cls(data=dcf)
+
+    def as_operator(self) -> DensityCompensationOp:
+        """Create a density compensation operator using a copy of the DCF."""
+        from mrpro.operators._DensityCompensationOp import DensityCompensationOp
+
+        return DensityCompensationOp(self.data)
```

Comment:
Would it make sense to have here `DensityCompensationOp(self)` because of 

_# TODO: change if einsumop can handle dataclasses_

in the DensityCompensationOp?

Reply:
I changed to self.data.clone()
to ensure that it is a copy, matching the docstring.

If we later on change the Operator, we will have to change this, yes, and decide if we want to create a copy or not..